.eslintrc.js

```diff
@@ -11,15 +11,20 @@ module.exports = {
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "prettier/@typescript-eslint",
+    "prettier",
   ],
   rules: {
     ...jsConfig.rules,
+    "@typescript-eslint/no-unsafe-argument": "off",
```

Comment:
This appears to be a new rule that didn't exist before. We had 123 violations which is too much to fix or disable by line.